### PR TITLE
Add Grafana resources

### PR DESCRIPTION
**Pull request by sarahbx**
##### Short description:

Add Grafana Resources for grafana-operator
https://github.com/grafana-operator/grafana-operator/

### CODE REVIEW

File: ocp_resources/grafana.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class Grafana(NamespacedResource):
```
Comment (myakove) [inline ocp_resources/grafana.py:4]: Empty class without to_dict implement required is not allowed.
Please introduce to_dict().

File: ocp_resources/grafana_data_source.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class GrafanaDataSource(NamespacedResource):
```
Comment (myakove) [inline ocp_resources/grafana_data_source.py:4]: Empty class without to_dict implement required is not allowed.
Please introduce to_dict().

File: ocp_resources/grafana_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                namespace="grafana",
+                label={"app": "grafana"},
+                configmap_name="my_dashboard_configmap",
+                configmap_key="dashboard_1",
+                data_sources=[{"inputName": "DS_PROMETHEUS", "datasourceName": "Prometheus"}]
+            )
+
+        """
+        super().__init__(name=name, namespace=namespace, yaml_file=yaml_file, **kwargs)
+        self.label = label
```
Comment (myakove) [inline ocp_resources/grafana_dashboard.py:46]: All of those required? Please add verification for required args under `if not self.yaml_file:`

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by sarahbx** [comment]
on hold due to recent release of v5 which altered the specs of the resources.

**Comment by myakove** [comment]
test

**Pull request closed without merging**